### PR TITLE
Adding more for conda environments

### PR DESCRIPTION
**Pull request by kraigb**

### CODE REVIEW

File: docs/languages/python.md [hunk 1]
```diff
@@ -4,20 +4,20 @@ Area: languages
...
 ContentId: c2cb770d-571d-4edf-9eb9-b5b8977c21a0
 PageTitle: Python in Visual Studio Code
-DateApproved: 02/01/2018
+DateApproved: 02/14/2018
 MetaDescription: Learn about Visual Studio Code features (code completion, debugging, snippets, linting) for Python.
 ---
 # Python in Visual Studio Code
 
-Working with Python in Visual Studio Code, using the [Microsoft Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python), is simple, fun, and productive. The extension leverages all of VS Code's power to provide auto complete and IntelliSense, linting, debugging, and unit testing, along with control over your Python environments.
+Working with Python in Visual Studio Code, using the [Microsoft Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python), is simple, fun, and productive. The extension works with all your Python environments, including Anaconda. It leverages all of VS Code's power to provide auto complete and IntelliSense, linting, debugging, and unit testing, along with the ability to easily switch between Python environments, including virtual and conda environments.
```
Comment (jflam) [inline docs/languages/python.md]: probably should be "conda environments"?

Comment (kraigb) [reply docs/languages/python.md]: Do you mean the word "Anaconda"? I did want to get that word in there somewhere...

**Comment by DonJayamanne** [comment]
Thanks for covering the edge cases mentioned in the meeting. 👍

**Comment by kraigb** [comment]
Certainly! I'm glad we had the chance to talk through them...that really helped.

**Pull request merged**